components/ThemeSwitch/index.js

```diff
@@ -1,36 +1,30 @@
-import { useTheme } from 'next-themes'
 import { useState, useEffect } from 'react'
 import { MoonIcon, SunIcon } from '@heroicons/react/solid'
+import { useTheme as useNextTheme } from 'next-themes'
+import { Switch, useTheme } from '@nextui-org/react'
+
+import { CgMoon } from 'react-icons/cg'
 
 export default function ThemeSwitch() {
   const [mounted, setMounted] = useState(false)
-  const { theme, setTheme } = useTheme()
+  const { theme } = useTheme();
 
   useEffect(() => {
     setMounted(true)
   }, [])
 
   if (!mounted) return null
 
-  return (
-    <button
-      data-testid="ThemeSwitch"
-      onClick={() => setTheme(theme === 'dark' ? 'light' : 'dark')}
-      className="text-white dark:text-white-100 dark:text-black cursor-pointer rounded-full border-transparent bg-transparent p-2"
-    >
-      {theme === 'light' ? (
-        <MoonIcon
-          data-testid="dark"
-          id="change-to-dark-theme"
-          className="h-5 w-5 text-primary-300"
-        />
-      ) : (
-        <SunIcon
-          data-testid="light"
-          id="change-to-light-theme"
-          className="h-5 w-5 text-primary-300"
-        />
-      )}
-    </button>
-  )
+    const { setTheme } = useNextTheme();
+    const { isDark, type } = useTheme();
+
+  return <Switch 
+  shadow 
```

Comment:
lost datatest Id

Reply:
I change the icon libary and need recreate this datatest and test